src/app/pages/dashboard/components/test-centre-journal-card/test-centre-journal-card.scss

```diff
@@ -0,0 +1,54 @@
+.test-centre-journal-button {
```

Comment:
youve added this button css here. could it be moved to the themes/button css file and given a generic name so we can use it for other blue buttons. also needs an entry for activated.
  --background-activated: var(--gds-dark-green-2);

Reply:
this is actually an additional set of styles added onto the styles for the actual css button which is located in the exact file that you just mentioned, the other dashboard buttons follow the same method of adding additional padding around it at this level of css, agree it could be renamed to something more generic, as i mentioned in chat open to any suggestions